diff --git a/indra/llrender/llgl.h b/indra/llrender/llgl.h
index 586a3fa..d0ccf21 100644
--- a/indra/llrender/llgl.h
+++ b/indra/llrender/llgl.h
@@ -355,6 +355,20 @@ private:
     LLGLStateStaticData LLGLState<state>::staticData = {#state, state, value, 0, nullptr, disabler_ptr}; \
        bool registered_##state = LLGLStateValidator::registerStateData(LLGLState<state>::staticData);

+template <>
+class LLGLState<0>
+{
+       LLGLState(S8 newState = 0) { }
+       virtual ~LLGLState() { }
+       virtual void enable() { }
+       virtual void disable() { }
+       static bool isEnabled() { return false; }
+       // For assertions. If feature is on or unsupported, return true.
+       static bool checkEnabled() { return true; }
+       // For assertions. If feature is off or unsupported, return true.
+       static bool checkDisabled() { return true; }
+};
+
 template <LLGLenum state>
 struct LLGLEnable : public LLGLState<state>
 {
